src/storage/rdb.cc

```diff
@@ -1005,3 +1000,37 @@ Status RDB::rdbSaveBinaryDoubleValue(double val) {
   memrev64ifbe(&val);
   return stream_->Write((const char *)(&val), sizeof(val));
 }
+
+Status RDB::rdbSaveZipListObject(const std::string &elem) {
+  // calc total ziplist size
+  uint prevlen = 0;
+  const size_t ziplist_size = zlHeaderSize + zlEndSize + elem.length() +
+                              ZipList::zipStorePrevEntryLength(nullptr, prevlen) +
+                              ZipList::zipStoreEntryEncoding(nullptr, elem.length());
+  auto zl = new unsigned char[ziplist_size];
```

Comment:
why new explicitly here?

Reply:
I have changed the code from using `new` directly to using `std::make_unique` for memory management. 
Thank you for your review and suggestions!